src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -36,6 +61,8 @@
 
    public Tag(String tagName, boolean logDuplicates) {
       eWONTagName = tagName;
+      tagDataFileName += tagName+"TagQueue.txt";

```

Comment:
This is more personal preference, but if you're defining something in the constructor, I don't think it should be defined in the header above. It makes it hard to follow. 

For example, in the header,
private String tagDataFileName;

and in the constructor,
tagDataFileName = "/usr/internalTagQueue/" + tagName + "TagQueue.txt"